packages/frontend/src/components/common/MonthAndYearInput.tsx

```diff
@@ -12,11 +12,7 @@ type Props = {
 const months = moment.months();
 
 const MonthAndYearInput: FC<Props> = ({ value, onChange, disabled }) => {
-    //Filtering a dimension returns a date, but filtering on a table returns a string on UTC
-    const utcMonthValue =
-        value instanceof Date
-            ? moment(value).month()
-            : moment(value).utc().month();
+    const utcMonthValue = moment(value).month();
```

Comment:
did you change these functions because the new tests caught a bug?

Reply:
Yep, this happened on my last timezone PR change . 

Before, the filters were returning a date or a filter, but after my last change, it always returns a string now.